_posts/2023-06-07-airflow-upgrade-to-2-5-3.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: "Airflow upgrade from 2.2.0 to 2.5.3 journey"
+author: dimonchik-suvorov
+tags:
+- featured
+- airflow-series
+- aws
+  team: Data Platform
+---
+
+Let me continue QP's `airflow-series` rubric and tell you a story of one Airflow upgrade. In this post I'm going to describe what infrastructure we had before, how we improved it and what difficulties we faced along the way so let's go!
+
+### At the beginning there were...
+
+Main parts of our Airflow infrastructure:
```

Comment:
There is gonna be a high-level diagram of our infrastructure so people will understand it better and keep this image in minds while they will be reading